zbus/src/address/address_list.rs

```diff
@@ -0,0 +1,80 @@
+use std::{borrow::Cow, fmt};
+
+use super::{Address, Error, Result, ToAddresses};
+
+/// A bus address list.
```

Comment:
I'm getting a deja-vu asking this but where does this concept come from and (more importantly) where is it used? I am asking because I am unable to find any reference to any such concept in the specification.

Reply:
In any case, it would be good to document it here with link(s) to appropriate docs.

Reply:
When connecting to a server, multiple server addresses can be separated by a semi-colon. The library will then try to connect to the first address and if that fails, it'll try to connect to the next one specified, and so forth. For example

unix:path=/tmp/dbus-test;unix:path=/tmp/dbus-test2

Reply:
The most notable examples are related to autolaunch, typically an address is given and if it doesn't exist a bus is spawned. Of course there are other usages, for fallback handling, especially in network environment. This is not typical afaict.

Reply:
I understand how it can be used or useful. I'm asking for actual uses of it. Does anyone ever use it? The API seems designed around supporting this so this is an important question IMO.

Reply:
OTOH, most of it the address API is sort of niche anyway as we'll have very few direct users of it. So maybe this is not such an important question.

Reply:
Configurabiliy of dbus server address has always been kind of niche. 99% of the case, you connect to a hardcoded system path :)